src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator.java

```diff
@@ -609,8 +616,21 @@ public ListBoxModel doFillServerUrlItems() {
             return BitbucketEndpointConfiguration.get().getEndpointItems();
         }
 
-        public FormValidation doCheckCredentialsId(@QueryParameter String value) {
+        public FormValidation doCheckCredentialsId(@AncestorInPath SCMSourceOwner context, @QueryParameter String serverUrl, @QueryParameter String value) {
```

Comment:
Are DescriptorImpl classes considered part of the public API that this plugin is exposing?

Reply:
no